app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,10 @@
 <resources>
     <string name="app_name">ComposeEcommerce</string>
-</resources>
\ No newline at end of file
+
+    <string name="home_discover">Discover</string>
+    <string name="home_suggested_for_you">SUGGESTED FOR YOU</string>
+    <string name="home_collection">Poly Cube 2021 Collection</string>
+    <string name="home_product_name">Poly Cubes</string>
+    <string name="home_product_price">From ฿8,000</string>
+    <string name="home_action_shop">SHOP</string>
```

Comment:
how about using textallcaps to ensure it in programmatically way https://tedblob.com/jetpack-compose-text-all-caps/?

Reply:
@hoangnguyen92dn ok, not a big deal